PyVALION/plotting.py

```diff
@@ -192,3 +193,131 @@ def plot_individual_mean_residuals(res_ion,
         plt.savefig(figname, bbox_inches='tight', facecolor='white')
         logger.info('Figure Ionosonde Mean Residuals is saved at: ', figname)
     return
+
+
+# -----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
+def plot_TEC_residuals_map(alat,
+                           alon,
+                           residuals,
+                           dtime,
+                           save_option=False,
+                           save_dir='',
+                           plot_name='TEC_Residuals_Map'):
+    """Plot residual map for TEC.
+
+    Parameters
+    ----------
+    alon : array-like (float)
+        Flattened array of longitudes in degrees.
+    alat : array-like (float)
+        Flattened array of latitudes in degrees.
+    residuals : dict
+        Dictionary with TEC residuals.
+    model_units: dict
+        Dictionary with units for each key in residuals.
+    dtime : datetime.datetime
+        Datetime of the validation day.
+    plot_dir : str
+        Direction where to save the figure.
+    plot_name : str
+        Output name, without directory, for the saved figure
+        (default='TEC_Residuals_Map.pdf').
+    """
+
+    projection1 = ccrs.PlateCarree()
+    fig, ax = plt.subplots(1, 1, sharex=True, sharey=True, figsize=(9, 3),
+                           constrained_layout=True,
+                           subplot_kw={'projection': projection1})
+    plt.xlim([-180, 180])
+    plt.ylim([-90, 90])
+    ax_plot = ax
+    ax_plot.set_xticks(np.arange(-180, 180 + 45, 90), crs=ccrs.PlateCarree())
+    ax_plot.set_yticks(np.arange(-90, 90 + 45, 45), crs=ccrs.PlateCarree())
+    ax_plot.set_xlabel('Geo Lon (°)')
+    ax_plot.set_ylabel('Geo Lat (°)')
+    ax_plot.set_facecolor('lightgray')
+    lon_formatter = cticker.LongitudeFormatter()
+    lat_formatter = cticker.LatitudeFormatter()
+    ax_plot.xaxis.set_major_formatter(lon_formatter)
+    ax_plot.yaxis.set_major_formatter(lat_formatter)
+    vmax = 30
+    norm = colors.Normalize(vmin=-vmax, vmax=vmax)
+    sc = ax_plot.scatter(alon, alat, c=residuals['TEC'], s=3,
+                         cmap='seismic', norm=norm)
+    ax_plot.coastlines(lw=0.5, color='black', zorder=1)
+    if np.size(dtime) > 1:
+        ax_plot.set_title('Residuals, ' + dtime[0].strftime('%Y%m%d') + ' - '
+                          + dtime[-1].strftime('%Y%m%d'))
+    else:
+        ax_plot.set_title('Residuals, ' + dtime.strftime('%Y%m%d'))
+    cbar = fig.colorbar(sc, ax=ax_plot)
+    cbar.set_label('TEC Residuals (TECU)')
+
+    if save_option:
+        # Save figure
+        figname = os.path.join(save_dir, plot_name + '.pdf')
+        plt.savefig(figname, bbox_inches='tight', facecolor='white')
+        print('Figure Residual Map is saved at: ', figname)
+    return
```

Comment:
```suggestion
    return fig
```

Recommend returning a figure handle, so that users can customize the figure further after creation.

Reply:
Applied this suggestion in both function my local branch. Updated docstring, tutorials, and example.rst. Commited changes!